vars/simpleMavenLibPipeline.groovy

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/groovy
+
+// TODO: Is body needed or can it be removed?
+def call(Map args, Closure body) {
```

Comment:
body trengs ikke siden vi ikke wrapper noe

Reply:
Slenge gjerne på en dokumentar her som er forsøkt på de andre stedene. Det som bør beskrives her er krav til pom-filen tenker jeg. Kanskje gi et minimalistisk eksempel for hvilke ting som må være satt i pom (altså ikke en full pom-fil, bare linjene som er relevant, f.eks. for `major-version`)?

Reply:
Har lagt ut første utkast på dok. På rett spor?

Reply:
Synes det ble bra!